core/src/main/resources/defaults/clustertemplates/7.2.15/aws/rt-datamart.json

```diff
@@ -99,8 +99,8 @@
           "attachedVolumes": [
             {
               "count": 2,
-              "size": 1900,
-              "type": "ephemeral"
+              "size": 1000,
+              "type": "standard"
```

Comment:
If "standard" in the template equates to EBS HDD (st1 or sc1) this is a bad choice.  EBS HDD has low IOPS and bad performance cliffs when the burst balance is exceeded.  gp3 (if supported) or some other SSD with guaranteed IOPS (no burst balance) should be used if we're going to move away from ephemeral.

Reply:
Now, I have changed it to gp2 as it is supported.